LICENSE.txt

```diff
@@ -205,13 +205,13 @@
 vector/src/main/java/org/apache/arrow/vector/util/IntObjectHashMap.java
 vector/src/main/java/org/apache/arrow/vector/util/IntObjectMap.java
 
-These file are derived from code from Netty, which is made available under the
+These file are derived from code from Netty(4.1.117), which is made available under the
```

Comment:
Could you add how to determine this (and other changes such as Guava 33.4.0 and the license header) to the PR description? e.g. commit ID you referred.
It's for easy to trace. We'll use the PR description for commit message.

Reply:
Done, I also updated my commit log：）

Reply:
Thanks.